File: itx-java/itx-worker/src/main/java/io/github/chehsunliu/itx/worker/compute/ComputeDispatcher.java
package io.github.chehsunliu.itx.worker.compute;

import io.github.chehsunliu.itx.contract.queue.MessageHandler;
import lombok.extern.slf4j.Slf4j;

@Slf4j
public final class ComputeDispatcher implements MessageHandler {

    @Override
    public void handle(String body) {
        // No compute-plane message types yet — just log and ack so the queue stays drained.
        log.info("compute message received (no handler yet) body={}", body);
    }
}

File: itx-kt/itx-worker/src/main/kotlin/io/github/chehsunliu/itx/worker/compute/ComputeDispatcher.kt
package io.github.chehsunliu.itx.worker.compute

import io.github.chehsunliu.itx.contract.queue.MessageHandler
import org.slf4j.LoggerFactory

class ComputeDispatcher : MessageHandler {
    private val log = LoggerFactory.getLogger(ComputeDispatcher::class.java)

    override suspend fun handle(body: String) {
        // No compute-plane message types yet — just log and ack so the queue stays drained.
        log.info("compute message received (no handler yet) body={}", body)
    }
}
